packages/graphback-cli/src/helpers/transformOpenApiSpec.ts

```diff
@@ -79,3 +45,37 @@ async function processSingleDefinition(model: string, isYaml: boolean, openApiCo
         logInfo(`   Failed to process OpenAPI definition: ${model}. Error: ${err}`);
     }
 }
+
+/**
+ * OpenAPI command line tool
+ * 1. Reads all the OpenAPI yaml and json files from the model folder
+ * 2. Generates GraphQL Schema
+ * 3. Renames original OpenAPI definitions so they will not be processed anymore
+ */
+export const transformOpenApiSpec = async () => {
```

Comment:
Was this some kind of rule that was failing? Order of functions

Reply:
Eslint complained that the function shouldn't be used before it is defined